File: INF/24/24_1/24_1.cpp
#include <iostream>
#include <string>
#include <fstream>
using namespace std;
 
main() {
    ifstream f;
    f.open("24_1.txt");
    string s;
    int ka, ke, count = 0;
    while(!f.eof()) {
        getline(f, s);
        ka = 0;
        ke = 0;
        for(int i = 0; i < s.size(); i++) {
            if(s[i] == 'A')
                ka++;
            if(s[i] == 'E')
                ke++;
        }
        if(ke > ka)
            count++;
    }
    cout << count;
    f.close();
}

File: 17.05/24/24-1.py
with open("24-1.txt") as f:
    count = 0
    lst = []
    for line in f:
        for i in line.split():
            lst.append(i)
    for i in range(len(lst)):
        if lst[i] == 'C' and lst[i + 1] == 'B' and lst[i + 3] == "B" and lst[i + 4] == "C":
            if lst[i + 2] != "A" and lst[i + 2] != "B" and lst[i + 2] != "F":
                
                count += 1
